server/setup.py

```diff
@@ -45,7 +45,7 @@
         'ftfy',
         'torch',
         'regex',
-        'torchvision',
+        'torchvision<=0.13.0' if sys.version_info <= (3, 7, 2) else 'torchvision',
```

Comment:
```suggestion
        'torchvision<=0.13.0' if sys.version_info[:2] <= (3, 7) else 'torchvision',
```

Reply:
what's the difference between these two?

Reply:
python 3.7.x: x ∈ [0, 15]

Reply:
https://www.python.org/downloads/